ospo-book/chapters/00-chapter.md

```diff
@@ -0,0 +1,27 @@
+# What is this book About?
+
+This book provides a guide to Open Source Programs Offices (OSPOs), to better understand their role in organizations, and the best practices for 
+establishing and managing an OSPO. The book covers a wide range of topics, including the reasons why organizations may need (or not) an OSPO, the 
+role of OSPOs in different types of organizations (public, private, small, medium, large), the challenges, antipatterns associated with OSPOs, and the 
+tools and processes required to establish and manage an effective OSPO. The book also does a brief introduction to assess an organization's readiness 
+for OSPO. As well as how to’s recommendations: from how to develop open source policies to how to engage with external open source communities.
+
+# What’s not in this book
+
+This book is not intended to be a guide to open source software development, nor does it cover the technical aspects of open source software development 
+in depth. Instead, the book focuses on the organizational and management aspects of open source dynamics within organizations by establishing OSPO roles. 
```

Comment:
I'm not sure what was meant on " by establishing OSPO roles" and I do not think that it is needed. 
```suggestion
in depth. Instead, the book focuses on the organizational and management aspects of open source dynamics within organizations. 
```

Reply:
```suggestion
in depth.
```